cve_bin_tool/cvedb.py

```diff
@@ -1173,8 +1177,18 @@ def fetch_from_mirror(self, mirror, pubkey, ignore_signature, log_signature_erro
 
     @contextlib.contextmanager
     def with_cursor(self):
+        """Context manager for database cursor."""
         cursor = self.db_open_and_get_cursor()
         try:
             yield cursor
         finally:
             self.db_close()
+
+    def ensure_unknown_metric(self, cursor):
```

Comment:
Why not moving this to `populate_metrics` and ensure that the table `metrics` gets also monitored for changes like in example `cve_range` in `get_cvelist_if_stale`.